### ISSUE TITLE
First arg expansion with long expression in 2nd arg

### ISSUE THREAD

**Issue by caesarsol** [prettier/prettier#2456]
This bug seems to happen only with a LogicalExpression chain inside a FunctionCall with an ArrowFunctionExpression and BlockStatement as the first argument.

More test cases in this [REPL example](https://prettier.io/playground/#%7B%22content%22%3A%22%2F%2F%20Correct%5Cnvar%20a%20%3D%20err.message.includes(%5C%22asd%5C%22)%20%26%26%20err.message.includes(%5C%22id%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes('%5C%221%5C%22')%20%26%26%20err.message.includes(%5C%22Model%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes(%5C%22%2Fid%5C%22)%20%26%26%20err.message.includes(%5C%22identifier(number)%5C%22)%5Cn%5Cn%2F%2F%20Correct%5Cnf(err.message.includes(%5C%22asd%5C%22)%20%26%26%20err.message.includes(%5C%22id%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes('%5C%221%5C%22')%20%26%26%20err.message.includes(%5C%22Model%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes(%5C%22%2Fid%5C%22)%20%26%26%20err.message.includes(%5C%22identifier(number)%5C%22)%20)%5Cn%5Cn%2F%2F%20Correct%5Cnf(x%2C%20y%2C%20err.message.includes(%5C%22asd%5C%22)%20%26%26%20err.message.includes(%5C%22id%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes('%5C%221%5C%22')%20%26%26%20err.message.includes(%5C%22Model%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes(%5C%22%2Fid%5C%22)%20%26%26%20err.message.includes(%5C%22identifier(number)%5C%22)%2Cz%2Czz)%5Cn%5Cn%2F%2F%20Correct%5Cnf(x%20%3D%3E%20y%2C%20err.message.includes(%5C%22asd%5C%22)%20%26%26%20err.message.includes(%5C%22id%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes('%5C%221%5C%22')%20%26%26%20err.message.includes(%5C%22Model%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes(%5C%22%2Fid%5C%22)%20%26%26%20err.message.includes(%5C%22identifier(number)%5C%22))%5Cn%5Cn%2F%2F%20Correct%5Cnf(function%20x()%20%7B%20y%20%7D%2C%20err.message.includes(%5C%22asd%5C%22)%20%26%26%20err.message.includes(%5C%22id%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes('%5C%221%5C%22')%20%26%26%20err.message.includes(%5C%22Model%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes(%5C%22%2Fid%5C%22)%20%26%26%20err.message.includes(%5C%22identifier(number)%5C%22))%5Cn%5Cn%2F%2F%20Correct%5Cnf(x%20%3D%3E%20%7B%7D%2C%20err.message.includes(%5C%22asd%5C%22)%20%26%26%20err.message.includes(%5C%22id%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes('%5C%221%5C%22')%20%26%26%20err.message.includes(%5C%22Model%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes(%5C%22%2Fid%5C%22)%20%26%26%20err.message.includes(%5C%22identifier(number)%5C%22))%5Cn%5Cn%2F%2F%20Incorrect!!!!!!!!!!!!!!!!!!!!!!!!!!%5Cnf(x%20%3D%3E%20%7B%20y%20%7D%2C%20err.message.includes(%5C%22asd%5C%22)%20%26%26%20err.message.includes(%5C%22id%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes('%5C%221%5C%22')%20%26%26%20err.message.includes(%5C%22Model%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes(%5C%22%2Fid%5C%22)%20%26%26%20err.message.includes(%5C%22identifier(number)%5C%22))%5Cn%5Cn%5Cn%5Cn%2F%2F%20Mysteriously%20correct%20because%20of%20the%20comment%5Cnf(%5Cn%20%20%2F%2F%20a%20comment%5Cn%20%20x%20%3D%3E%20%7B%20y%20%7D%2C%20%5Cn%20%20err.message.includes(%5C%22asd%5C%22)%20%26%26%20err.message.includes(%5C%22id%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes('%5C%221%5C%22')%20%26%26%20err.message.includes(%5C%22Model%5C%22)%20%26%26%20%5Cn%20%20%20%20%20err.message.includes(%5C%22%2Fid%5C%22)%20%26%26%20err.message.includes(%5C%22identifier(number)%5C%22))%22%2C%22options%22%3A%7B%22printWidth%22%3A100%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%2C%22ast%22%3Atrue%7D%7D).

```js
// INPUT
f(x => { y }, err.message.includes("asd") && err.message.includes("id") && 
     err.message.includes('"1"') && err.message.includes("Model") && 
     err.message.includes("/id") && err.message.includes("identifier(number)")
)

// OUTPUT
f(x => {
  y;
}, err.message.includes("asd") && err.message.includes("id") && err.message.includes('"1"') && err.message.includes("Model") && err.message.includes("/id") && err.message.includes("identifier(number)"));

// EXPECTED
f(
  x => {
    y;
  },
  err.message.includes("asd") &&
    err.message.includes("id") &&
    err.message.includes('"1"') &&
    err.message.includes("Model") &&
    err.message.includes("/id") &&
    err.message.includes("identifier(number)")
);
```

But mysteriously fixed if I add a comment inside the function:

```js
// INPUT
f(
  // a comment
  x => { y }, 
  err.message.includes("asd") && err.message.includes("id") && 
     err.message.includes('"1"') && err.message.includes("Model") && 
     err.message.includes("/id") && err.message.includes("identifier(number)")
)

// OUTPUT
f(
  // a comment
  x => {
    y;
  },
  err.message.includes("asd") &&
    err.message.includes("id") &&
    err.message.includes('"1"') &&
    err.message.includes("Model") &&
    err.message.includes("/id") &&
    err.message.includes("identifier(number)")
);
```

**Comment by vjeux** [comment]
Wow, this looks bad. I'm not really sure what's going on here but it's definitely not intended.

**Comment by azz** [comment]
Introduced way back in #947. Kind of tricky to fix properly.

**Comment by caesarsol** [comment]
Could the very specific case of #947 signaled by @azz be generalized to function calls whose second argument exceeds the chars limit?

**Comment by caesarsol** [comment]
Sorry, I mean **does not** exceeds

**Comment by thorn0** [comment]
fixed by #13341
